blackbox/docs/enterprise/monitoring.txt

```diff
@@ -0,0 +1,98 @@
+.. _jmx_monitoring_plugin:
+
+=====================
+JMX Monitoring Plugin
+=====================
+
+The CrateDB monitoring plugin exposes query metrics via the `JMX`_ API.
+
+This plugin is an `enterprise`_ feature.
+
+Enabling the JMX API
+--------------------
+
+.. highlight:: none
+
+To monitor CrateDB using the JMX API, you must to set the following system properties before you start CrateDB::
```

Comment:
I can't link to the config because this branch doesn't have the changes made to the config section that include the enterprise setting. could you merge that stuff into this branch and I can update my branch, @kovrus? (or how do you want to do it?